ApiDemos/kotlin/app/src/main/java/com/example/kotlindemos/PolylineDemoActivity.kt

```diff
@@ -0,0 +1,291 @@
+import com.google.android.gms.maps.model.CustomCap
+import com.google.android.gms.maps.model.Dash
+import com.google.android.gms.maps.model.Dot
+import com.google.android.gms.maps.model.Gap
+import com.google.android.gms.maps.model.JointType
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.PatternItem
+import com.google.android.gms.maps.model.Polyline
+import com.google.android.gms.maps.model.PolylineOptions
+import com.google.android.gms.maps.model.RoundCap
+import com.google.android.gms.maps.model.SquareCap
+import android.graphics.Color
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.view.View
+import android.widget.AdapterView
+import android.widget.ArrayAdapter
+import android.widget.CheckBox
+import android.widget.SeekBar
+import android.widget.Spinner
+import java.util.Arrays
+
+/**
+ * This shows how to draw polylines on a map.
+ */
+class PolylineDemoActivity :
+        AppCompatActivity(),
+        OnMapReadyCallback,
+        SeekBar.OnSeekBarChangeListener,
+        AdapterView.OnItemSelectedListener {
+
+    private val CUSTOM_CAP_IMAGE_REF_WIDTH_PX = 50
+    private val INITIAL_STROKE_WIDTH_PX = 5
+    private val MAX_WIDTH_PX = 100
+    private val MAX_HUE_DEGREES = 360
+    private val MAX_ALPHA = 255
+    private val PATTERN_DASH_LENGTH_PX = 50
+    private val PATTERN_GAP_LENGTH_PX = 20
+
+    // City locations for mutable polyline.
+    private val adelaideLatLng = LatLng(-34.92873, 138.59995)
+    private val darwinLatLng = LatLng(-12.4258647, 130.7932231)
+    private val melbourneLatLng = LatLng(-37.81319, 144.96298)
+    private val perthLatLng = LatLng(-31.95285, 115.85734)
+
+    // Airport locations for geodesic polyline.
+    private val aklLatLng = LatLng(-37.006254, 174.783018)
+    private val jfkLatLng = LatLng(40.641051, -73.777485)
+    private val laxLatLng = LatLng(33.936524, -118.377686)
+    private val lhrLatLng = LatLng(51.471547, -0.460052)
+
+    private val dot = Dot()
+    private val dash = Dash(PATTERN_DASH_LENGTH_PX.toFloat())
+    private val gap = Gap(PATTERN_GAP_LENGTH_PX.toFloat())
+    private val patternDotted = Arrays.asList(dot, gap)
+    private val patternDashed = Arrays.asList(dash, gap)
+    private val patternMixed = Arrays.asList(dot, gap, dot, dash, gap)
+
+    private lateinit var mutablePolyline: Polyline
+    private lateinit var hueBar: SeekBar
+    private lateinit var alphaBar: SeekBar
+    private lateinit var widthBar: SeekBar
+    private lateinit var startCapSpinner: Spinner
+    private lateinit var endCapSpinner: Spinner
+    private lateinit var jointTypeSpinner: Spinner
+    private lateinit var patternSpinner: Spinner
+    private lateinit var clickabilityCheckbox: CheckBox
+
+    // These are the options for polyline caps, joints and patterns. We use their
+    // string resource IDs as identifiers.
+    private val capTypeNameResourceIds = intArrayOf(R.string.cap_butt, // Default
```

Comment:
I'm not sure if you need the "Default" comment there

Reply:
Yep, I've got rid of all three default comments.